slackbridge/bots.py

```diff
@@ -62,8 +67,8 @@ def resolve(self):
                 return
 
             channel_id = self.raw_message['channel']
-            if channel_id in self.channels:
-                channel_name = self.channels[channel_id]['name']
+            if channel_id in IRCBot.channels.keys():
```

Comment:
What is the benefit of doing this versus:
```python
if channel_id in IRCBot.channels
```
That is the convention we've been following for this project.

Reply:
That would work too and is cleaner, thanks for catching that.